### DISCUSSION TITLE
files needed for run_lfric_atm_clim_gal9_chem-C12_spice_intel_fast-debug-64bit-crun0

### DISCUSSION THREAD

**Discussion by alanjhewitt** [MetOffice/simulation-systems#221] [LFRic]
I would like to get run_lfric_atm_clim_gal9_chem-C12_spice_intel_fast-debug-64bit-crun0 working from applications/lfric_atm/example/

I copied the contents of the spice run from /scratch into this directory

I got past the line "Physics fields collections initialised" but not as far as the next printed line line "Gungho: Initialising prognostic fields" and got message "Killed".

In the configuration.nml file that I copied, I have links to files that I am sure that I will need to copy into my example directory, e.g.
"easy_absorption_lw_ancil_path='cmip6_volcanic/clim_1850-2014/volc_aer_absorption_lw.nc.ugrid',"
there are also some files that have a long path, so I assume that these should work even if it would be better to have them in the example directory.
"checkpoint_stem_name='/home/h04/ahewitt/cylc-run/vn1.1_vanilla/run1/share/data/restart_lfric_atm_clim_gal9_chem-C12_spice_intel_fast-debug-64bit_tstep',"

I cannot seem to find e.g.  "volc_aer_absorption_lw.nc.ugrid" to be able to move it to the example directory.

Does anyone know where I can find this?

**Comment by stevemullerworth** [comment]
The pathnames of ancil files are prefixed with the `ancil_directory` setting in the configuration.nml file - so that's probably not an issue.

If you are sure you cannot find a stack trace, then it's possible that your job has run out of memory. Are you running on Spice? The suite may have requested more memory and will have specified `ulimit -s unlimited` (some jobs fail due to lack of stack space).

I'm not sure of the reason for copying the files to the `example` directory. A general process for copying rose stem run_* directories, including tweaks to pathnames for the checkpoint_stem_name and in the iodef.xml file, is described here:

https://code.metoffice.gov.uk/trac/lfric/wiki/LFRicTechnical/DebuggingTips#Usingotherconfigurations

I've just added information about setting `ulimit` having recently suffered from a job failure due to this issue.

**Reply by iboutle** [reply]
Yes, you shouldn't need to copy any of the ancil files anywhere - the command line run should just be able to pick these up directly from the ancil_directory. All you need to copy are the files actually in the work directory of cylc-run, i.e. the configuration namelist, the xml files, the mesh and the 3 folders containing input files for radaer/radiation.

It will also give a more helpful error message if it can't find a file, so if all you have is "killed" it's unlikely to be that. As Steve says, the suspicion would be this job might just be too big to run on a VDI - can you try running, e.g. the non-chem climate version, as I think I have run that okay before.
